tests/unit/test_utils.py

```diff
@@ -99,3 +98,24 @@ def test_flatten_list(value: List, anticpated_result: List) -> None:
     """test for flatten list"""
     actual_result = utils.flatten_list(value)
     assert list(actual_result) == anticpated_result
+
+
+@pytest.mark.parametrize(
+    "value, anticpated_result",
```

Comment:
For the params list, it's more readable to use a sequence of args, not a string with coma-separated names. When I have time, I'll integrate the corresponding flake8 plugin that catches this problem.
```suggestion
    ("value", "anticpated_result"),
```

Reply:
doing away with this.........................................